libraries/datahandler/src/main/thrift/components.thrift

```diff
@@ -480,13 +481,16 @@ struct ClearingReport{
 }
 
 struct ReleaseNode {
-     1: required string releaseId,
-     2: optional string releaseRelationship,
-     3: optional string mainlineState,
-     4: optional string comment,
-     5: optional string createOn,
-     6: optional string createBy,
-     7: optional list<ReleaseNode> releaseLink,
+    1: required string releaseId,
+    8: optional string releaseName, // This field is used to view data, is not stored in db
+    9: optional string releaseVersion, // This field is used to view data, is not stored in db
+    10: optional string componentId, // This field is used to view data, is not stored in db
+    2: optional string releaseRelationship,
```

Comment:
Please rearrange the fields as per their declaration order (8, 9, 10 after 7) :-)

Reply:
@GMishx thanks for your comment, I fixed it